_episodes/format.md

```diff
@@ -3,51 +3,260 @@ title: "File format"
+>> Also, the NetCDF format can contain both data and metadata.
+>>
+>> 2. Therefore, yes, the NetCDF format helps to make the data FAIR.
+> {: .solution}
+{: .challenge}
+
+- [GRIB](https://www.wmo.int/pages/prog/www/DPS/FM92-GRIB2-11-2003.pdf):
+  GRIB stands for general regularly-distributed information in binary.
+  It is **commonly used** by the [World Meteorological Organization](https://public.wmo.int/en)(WMO) for weather model data.
+
+  - GRIB coded data consist of a continuous bit-stream made of a sequence of
+  [octets](https://en.wikipedia.org/wiki/Octet_(computing)).
+  Thus, the representation of the data is **independent of** any particular machine representation.
+
+  - GRIB messages contain the **data** and **metadata**.
+
+> ## Why GRIB format
+>
+> You work with some geospatial data for a project in your community.
+> You choose the GRIB format for the data.
+> Your colleague asks:
+>
+> 1. Why did you choose the GRIB format?
+> 2. Does this format help to make the data FAIR?
+>
+>> ## Solution
+>>
+>> 1. You found that GRIB format is the standard in your research community and
+>> therefore interoperable and reusable for anyone.
+>> GRIB format is independent of any particular machine representation.
+>> Also, the GRIB format can contain both data and metadata.
+>>
+>> 2. Therefore, yes, the GRIB format helps to make the data FAIR.
+> {: .solution}
+{: .challenge}
+
+- [GeoTIFF](https://www.geospatialworld.net/article/geotiff-a-standard-image-file-format-for-gis-applications/):
+  GeoTIFF is a standard image file format to describe and store raster image data with geographic information.
+  So it can be used by  Geographic Information System (GIS) applications.
+
+  - GeoTIFF format is suitable for a **wide range of applications worldwide**.
+  For example, satellite imaging systems, scanned aerial photography, scanned maps,
+  digital elevation models, or as a result of geographic analyses.
+
+  - It is **independent** of a computer’s architecture, operating system, and graphics hardware.
+
+  - GeoTIFF format has a three-level hierarchy: file header, image file directories, and **data**.
+
+  - GeoTIFF format stores a **broad range of georeferencing information**, catering to geographic
+  as well as projected coordinate systems.
+
+> ## Why GeoTIFF format
+>
+> You work with some geospatial data for a project in your community.
+> You choose the GeoTIFF format for the data.
+> Your colleague asks:
+>
+> 1. Why did you choose the GeoTIFF format?
+> 2. Does this format help to make the data FAIR?
+>
+>> ## Solution
+>>
+>> 1. You found that GeoTIFF format is the standard in your research community and
+>> therefore interoperable and reusable for anyone.
+>> GeoTIFF format is independent of computer’s architecture, operating system, and graphics hardware.
+>> Also, the GeoTIFF format can contain both data and geographic information.
+>>
+>> 2. Therefore, yes, the GeoTIFF format helps to make the data FAIR.
```

Comment:
Maybe it would be nice to also show some examples of data that don't help to make the data FAIR. Aren't there like arcgis native file formats or so?

Reply:
I disagree to add formats that have too many disadvantages. It will be confusing. 
Actually, ArcGIS is one of the gis applications. GeoTIFF can be used by many gis applications.
I added an example of using the GeoTIFF format to the text.